stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -392,4 +392,13 @@ end
 
 end
 
+@testset "issue #33704, casting low-rank CholeskyPivoted to Matrix" begin
+    T = Float64
+    n = 8
+    A = randn(T, (n,1))
```

Comment:
`randn(8,1)` will return would you're asking here for, and neither `T` nor `n` are used again. So why not reduce the number of lines here?

Reply:
I made the change. Let me know if there is anything else.